release-calendar/src/server/app.ts

```diff
@@ -18,10 +18,14 @@ import {Connection, createConnection} from 'typeorm';
 import {RepositoryService} from './repository-service';
 import PromotionEntity from './entities/promotion';
 import ReleaseEntity from './entities/release';
+import bodyParser from 'body-parser';
 import express from 'express';
+import moment from 'moment';
 import path from 'path';
 import rateLimit from 'express-rate-limit';
 
+const _CTIME_FORMAT = 'ddd MMM D HH:mm:ss YYYY';
```

Comment:
Is it possible to use a more standard format here, like a Unix timestamp or the UTC format? I'm guessing this is related to the format the versioning subscriber deals with, but it seems like that formatting could be done on the request side rather than adjusting the API to that particular format. WDYT?

Reply:
Good point about not adjusting the API side. Ok, going to use ISO format then.

Reply:
SGTM